docs/nos-client/api.md

```diff
@@ -13,7 +13,78 @@ client, the API can be accessed via `window.NOS.V1`.  All functions return a
 user rejects the action.
 
 
+## Import and usage of the nOS API
+---
+
+## Vanilla Javascript
+```javascript
+// Contains all nOS API Functions
+const nos = window.NOS.V1;
+
+// Contains all exposed assets
+const assets = window.NOS.ASSETS;
+```
+
+## Installation and usage of API-Functions
+
+First, we need to install the [api-functions](https://www.npmjs.com/package/@nosplatform/api-functions) npm package. This is done with the command(s) below.
+
+```javascript
+// Using NPM
+npm i --save @nosplatform/api-functions
+
+// Using Yarn
+yarn add @nosplatform/api-functions
+```
+
+#### Usage api-functions inside React
+To use the nOS API inside a React package, we highly suggest using the HoC component and the nosProps for validation.
```

Comment:
It might be more clear to specify which HOC and link to the package here.  I know you already link to it above, but people often skip to the section they need, and they would be missing context if they jumped right to this.

> we highly recommend using the `injectNOS` HOC and `nosProps` propType from [api-functions](https://www.npmjs.com/package/@nosplatform/api-functions).

Reply:
Also a minor nitpick, I think the preferred capitalization is HOC per the [React documentation](https://reactjs.org/docs/higher-order-components.html).